ios/authentication.md

```diff
@@ -1014,6 +1014,14 @@ Note: your user pool domain is something like: `domain_prefix-<env-name>.auth.<r
 ![Image]({{common_media}}/cognitoHostedUI/amazon5.png)
 5. Choose Save.
 
+##### Setting up Hosted UI Domain with Apple
+
+1. The easiest way to get HostedUI configuration in your app is through automated set up with CLI. Choose a supported identity provider in your set up (Google and/or Facebook, or Login with Amazon) you would like to use. 
+
+2. Once you have the configuration for your hosted UI in your application, learn more about [How to set up Sign in with Apple for Amazon Cognito](https://aws.amazon.com/blogs/security/how-to-set-up-sign-in-with-apple-for-amazon-cognito/).
```

Comment:
Does this step elide a bunch of other steps that the user needs to pick from the blog post, a la "draw the rest of the owl?" If so, any value in hoisting those steps here?

Reply:
so some context here is that Amplify CLI does not support Apple sign in. Cognito does and our SDK does which is why we should have some steps in the docs.

missing steps i can think of that i can add is `amplify console auth` to open your UserPool in AWS Console. there might be a disconnect from the CLI to getting to their cognito user pool, which is why they need to run that command or navigate there (and then they need to know which userpool was provisioned) manually.

so i'm not against hoisting the steps here, it would increase the page size from the already really large page, so maybe we just need a separate page and link it from here. the other question is how long before CLI priotizies apple sign in? asking over in https://github.com/aws-amplify/amplify-cli/issues/3434 as that may influence our decision here. it looks like the blog was updated before, and so hoisting means we will also take ownership of cognito steps and updates, which is fair to do, as we already guide customers to amplify docs over cognito docs when using our SDK.

Reply:
Thanks for the context. I read the page and understand more fully now what this would entail. Assuming it's on the near term roadmap, my vote would be to wait until the CLI supports it directly and write the (much reduced) set of steps here.